velocity_controller_node:
  ros__parameters:
      Kp: 0.35
      Ki: 0.0
      max_integral_torque: 0.0
      histerisis_kickin_ms: 0.0
      histerisis_reset_ms: 0.0
      min_time_to_max_accel_sec: 0.5
